src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -36,7 +35,7 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
   val shutdownWait = 10.seconds
 
   //whatever happens we try to clean up
-  sys.addShutdownHook(shutdown)
+  sys.addShutdownHook(shutdown())
```

Comment:
Interesting, would this actually change behaviour?

Reply:
No, it doesn't change behavior. It's just because IntelliJ yielded warning about a parameterless call.